src/pudl/etl/epacems_assets.py

```diff
@@ -44,23 +44,33 @@ def hourly_emissions_epacems(
     epacems_settings = context.resources.dataset_settings.epacems
 
     schema = Resource.from_id("hourly_emissions_epacems").to_pyarrow()
-    epacems_path = (
+    partitioned_path = (
         Path(context.op_config["pudl_output_path"]) / "hourly_emissions_epacems"
     )
-    epacems_path.mkdir(exist_ok=True)
+    partitioned_path.mkdir(exist_ok=True)
```

Comment:
Ok after chatting with @zaneselvans and @bendnorman we decided to just write to the monolithic parquet file in the same place we are writing to the partitioned files. It's a little clunky, but I think this is definitely the simplest option